### DISCUSSION TITLE
Extracting code coverage metrics?

### DISCUSSION THREAD

**Discussion by jarthod** [rubycdp/ferrum#534] [Ideas]
Hi,

I just thought about the code codeverage tool in chrome and how awesome it would be to be able to execute it across a test suite and get the results. I found out that [puppeteer seems to support interacting with the code coverage tool](https://philkrie.me/2018/07/04/extracting-coverage.html) (starting it and collecting the result at least). And so I'm wondering if that could be a good idea for `ferrum` too? would it be complicated? can it already be done with `extensions` maybe (I don't think so).

Thanks for your input!

**Comment by route** [comment]
Hi @jarthod yes I had similar idea in the past, it's worth trying to implement it, maybe as a side project under rubycdp not sure. Anyways we are close to Ferrum and Cuprite 1.0 this is top priotity by now, after that I have a bunch of ideas to check including stealth mode and so on...

**Comment by sandstrom** [comment]
I'm doing some issue gardening 🌱🌿 🌷 and came upon this issue.

What do you think about moving some of the more exotic feature requests into discussions, to gauge interest? For any open-source project there will be a ton of ideas, but it rarely make sense to satisfy all of them (implementing and maintaining is a lot of work).

There should be a button 'move to discussions' in Github's UI.

By closing some old issues we reduce the list of open issues to a more manageable set.

**Comment by route** [comment]
Totally agree. Moved.
